test/e2e/play_kube_test.go

```diff
@@ -4506,4 +4544,91 @@ spec:
 		Expect(exec).Should(Exit(0))
 		Expect(exec.OutputToString()).Should(Equal("hi"))
 	})
+
+	It("podman play kube without Ports - curl should fail", func() {
+		err := writeYaml(publishPortsPodWithoutPorts, kubeYaml)
+		Expect(err).ToNot(HaveOccurred())
+
+		kube := podmanTest.Podman([]string{"play", "kube", kubeYaml})
+		kube.WaitWithDefaultTimeout()
+		Expect(kube).Should(Exit(0))
+
+		curlTest := podmanTest.Podman([]string{"run", "--network", "host", "quay.io/libpod/alpine_nginx:latest", "curl", "-s", "localhost:19000"})
```

Comment:
```suggestion
		curlTest := podmanTest.Podman([]string{"run", "--network", "host", NGINX, "curl", "-s", "localhost:19000"})
```

For other tests as well.  We have consts for the images such that we can quickly address flakes, do updates, etc.

Reply:
Funny thing is that I found this container image through the const, but forgot to use it. Fixed